stories/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+	"rules": {
```

Comment:
This indent is off

Reply:
hmm I thought I fixed that but I guess not 🤷‍♀️  will try again : )